pyhf/commandline.py

```diff
@@ -28,6 +29,18 @@ def xml2json(entrypoint_xml, basedir, output_file, track_progress):
         json.dump(spec, open(output_file, 'w+'), indent=4, sort_keys=True)
         log.info("Written to {0:s}".format(output_file))
 
+@pyhf.command()
+@click.argument('workspace', default = '-')
+@click.argument('xmlfile', default = '-')
```

Comment:
workspace and xmlfile can't both be stdin. How do you differentiate the two?

Reply:
so but they are both tty streams by default. Click differentiates between the two by the mode you open them with 

'r' -> stdin
'w' -> stdout